### DISCUSSION TITLE
What if Github pins have github-readme-stats internally?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#32036] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey there!

***Warning: This is just a random idea that I had.***
*What if *Github pins* have *github-readme-stats* internally? would it be possible?*

![image](https://user-images.githubusercontent.com/111701513/189827708-155fe31b-1383-4ee5-b5d2-e771a6ccd339.png)

***source:*** https://github.com/anuraghazra/github-readme-stats
***image-description:*** Image data is fictitious.

**Comment by jfullstackdev** [comment, marked as answer]
there is the Insight for every repo, you can use that through the API to create the same service like that, if you are really into it
